cli/cli.go

```diff
@@ -168,6 +169,9 @@ func preRun(cmd *cobra.Command, args []string) {
 	configuration.Init(configPath)
 	configFile := viper.ConfigFileUsed()
 
+	// initialize locale
+	i18n.SetLocale(viper.GetString("locale"))
```

Comment:
Shall we set locale explicitly?
Can we obtain the locale settings from system preferences (on mac, linux and windows) if not specified?